tests/test_manifest.py

```diff
@@ -60,16 +60,16 @@ def test_subset_fields():
     subset = Subset(specification=subset_spec, base_path="/tmp")
 
     # add a field
-    subset.add_field(name="data2", type_=Type.binary)
+    subset.add_field(name="data2", type_=Type("binary"))
```

Comment:
It's a bit unfortunate that we lose the ability to reference types by attribute (and therefore autocomplete, static analysis, etc., ...). Unfortunately I don't see a straightforward way to keep them either.

Reply:
yeah indeed, you would need to know/type all possible combinations which is not very feasible. But it's not user-facing so it should not have that big of an impact.